CHANGELOG.md

```diff
@@ -10,9 +10,9 @@
 * Fix location button in IE 11.
 * Fixed question mark icon.
 * Added text to last step of create activity if no image is selected.
-* Added drush translation commands to platform build
-* Added button to create activity for anonymous users
-* Added redirect for anonymous users for access denied page
+* Added drush translation commands to platform build.
+* Added button to create activity for anonymous users.
+* Added redirect for anonymous users for access denied page.
```

Comment:
Move changes from "In development" to "v1.0.4". Remember to leave "In development" empty.